src/views/preview/presentation.jsx

```diff
@@ -313,10 +315,21 @@ const PreviewPresentation = ({
                                     <h4>Comments</h4>
                                     {/* TODO: Add toggle comments component and logic*/}
                                 </FlexRow>
+
+                                <FlexRow className="comments-root-reply">
+                                    {isLoggedIn &&
```

Comment:
This will also need to toggle based on whether comments are turned on or off for the project. But there is no way for you to know that info atm (logged here https://github.com/LLK/scratch-api/issues/584)